js/interface.js

```diff
@@ -390,6 +392,39 @@ function trimColumns(columns) {
   });
 }
 
+function getEmptyColumns(columns, entries) {
+  var emptyColumns = _.filter(columns, function(column) {
+    return emptyColumnNameRegex.test(column);
+  });
+
+  _.forEach(entries, function(entry) {
+    if (!emptyColumns.length) {
```

Comment:
You'll want this outside the _.forEach loop too because if there is no empty column, there's no need to go through the _.forEach loop at all. The current code could end up going through this `if` check `entries.length` number of times.

Reply:
I added condition outside the loop and added comment for this row. We still need to check `emptyColumns.length` inside _.forEach loop because when all `emptyColumns` have values on `N` iteration (`N < entries.length`) we must stop iterating through entries.

Reply:
Yes, keeping it inside the loop makes sense too.